pkgs/data/icons/catppuccin-cursors/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, inkscape
+, xcursorgen
+, makeFontsConf
+}:
+
+let
+  dimensions = {
+    palette = [ "Frappe" "Latte" "Macchiato" "Mocha" ];
+    color = [ "Blue" "Dark" "Flamingo" "Green" "Lavender" "Light" "Maroon" "Mauve" "Peach" "Pink" "Red" "Rosewater" "Sapphire" "Sky" "Teal" "Yellow" ];
+  };
+  product = lib.attrsets.cartesianProductOfSets dimensions;
+  variantName = { palette, color }: (lib.strings.toLower palette) + color;
+  variants = map variantName product;
+in
+stdenvNoCC.mkDerivation rec {
+  pname = "catppuccin-cursors";
+  version = "unstable-2022-08-23";
+
+  src = fetchFromGitHub {
+    owner = "catppuccin";
+    repo = "cursors";
+    rev = "3d3023606939471c45cff7b643bffc5d5d4ff29c";
+    sha256 = "1z9cjxxsj3vrmhsw1k05b31zmncz1ksaswc4r1k3vd2mmpigq1nk";
+  };
+
+  outputs = variants
+    ++ [ "out" ]; # dummy "out" output to prevent breakage
+
+  outputsToInstall = [];
+
+  nativeBuildInputs = [
+    inkscape
+    xcursorgen
+  ];
+
+  postPatch = ''
+    patchShebangs .
```

Comment:
Might be nice to be more specific what needs to be patched, just like in the [comixcursors PR](https://github.com/NixOS/nixpkgs/pull/160069/commits/c2ea196a53f8703b76198c1d10aac4b529705b7c#diff-193be6e94eef9fae9d91f1f25778d1e7b96682219d5c86e6e6fa89035db190a7R36)